tests/aws/services/events/test_events_integrations.py

```diff
@@ -77,7 +84,6 @@ def test_put_events_with_target_sqs_new_region(aws_client_factory):
 
 
 @markers.aws.unknown
```

Comment:
not yet validated?

Reply:
fixed in: https://github.com/localstack/localstack/pull/10664/commits/d2b4586aebd3486090d03bc97244f06873ed1eed